transactions/read-only/parallel.md

```diff
@@ -23,112 +23,115 @@ Chain APIs can be classified into reads and writes.
+| get_accounts_by_authorizers     | medium_low  | none                                        | yes                    | yes                      |
+| get_transaction_status          | medium_low  | none                                        | yes                    | yes                      |
+| send_read_only_transaction      | low         | none                                        | yes                    | yes                      |
+| compute_transaction             | low         | main thread: temporally change chainbase             | no            | no                       |
+| push_block                      | medium_low  | main thread: chainbase, forkdb, producer, controller | no            | no                       |
+| push_transaction                | low         | main thread: chainbase, forkdb, producer, controller | no            | no                       |
+| push_transactions               | low         | main thread: chainbase, forkdb, producer, controller | no            | no                       |
+| send_transaction                | low         | main thread: chainbase, forkdb, producer, controller | no            | no                       |
+| send_transaction2               | low         | main thread: chainbase, forkdb, producer, controller | no            | no                       |
 
 
 ### Producer APIs
 They are received on the HTTP thread, processed on the main thread, and responses are sent on the HTTP thread.
 
-| API                                        | priority    |  Global data modified        | Safe to read-only trx? |
-|--------------------------------------------|-------------|------------------------------|------------------------|
-| pause                                      | medium_high | main thread: producer's \_pause_production  | yes     |
-| resume                                     | medium_high | main thread: producer's \_pause_production  | yes     |
-| paused                                     | medium_high |                              | yes                    |
+| API                                        | priority    |  Global data modified        | Safe to read-only trx? | Safe to run in parallel? |
+|--------------------------------------------|-------------|------------------------------|------------------------|--------------------------|
+| pause                                      | medium_high |                              | yes                    | no                       |
+| resume                                     | medium_high | calls abort_block() and schedule_production_loop()  | no  | no                   |
+| paused                                     | medium_high |                              | yes                    | yes                      |
 | get_runtime_options                        | medium_high |                              | yes                    |
```

Comment:
This is missing an entry in the `Safe to run in parallel?` column. I assume it should be `yes`.